src/lib/state.test.js

```diff
@@ -0,0 +1,57 @@
+import state from './state';
+
+describe('state', () => {
+  describe('updateTitle', () => {
```

Comment:
We should also make this file more flat

https://kentcdodds.com/blog/avoid-nesting-when-youre-testing